pim-tool-back/src/main/java/pilotproject/Project_Management_ELCA/model/entity/Project.java

```diff
@@ -0,0 +1,72 @@
+package pilotproject.Project_Management_ELCA.model.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Set;
+
+/**
+ * @author vlp
+ */
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Project {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne()
+    @JoinColumn(name = "group_id")
+    private Group group;
+
+    @Column(unique = true, nullable = false)
+    private int projectNumber;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String customer;
+
+    public enum Status {
```

Comment:
[S] This is duplicate with the Status in Dto. Both classes should share the enum.
[N] Please move this enum to its own file.